packages/magento-product-configurable/graphql/ConfiguredVariant.graphql

```diff
@@ -22,4 +13,10 @@ fragment ConfiguredVariant on SimpleProduct @injectable {
     }
     quantity
   }
+  url_rewrites {
+    url
+  }
+  ...ProductReviewsConfiguredVariant
+  ...ProductPageDescription
+  ...ProductPageMeta
```

Comment:
Might it be an idea to 'just' use the ProductPageItem fragment instead of the ConfiguredVariant and have everything work automatically?